### ISSUE TITLE
portable_mtime for py3_image

### ISSUE THREAD

**Issue by amir-f** [bazelbuild/rules_docker#2269]
# 🚀 feature request

### Description

When using `py3_image` the _mtime_ (modification time) for all the files are set to 0 unix time (cira 1970). Some python libraries panic when they see that (eg. `zipfile` python builtin lib `ValueError: ZIP does not support timestamps before 1980`)

### Describe the solution you'd like

`container_image` has an option to set all mtimes to a fixed valued: see [`portable_mtime`](https://github.com/bazelbuild/rules_docker/blob/8e70c6bcb584a15a8fd061ea489b933c0ff344ca/container/layer.bzl#L355) but IIUC it needs to be routed to `py3_image` through the signatures somehow, hence this feature request.

Basically can we by default or through an option in `py3_image` set all the mtimes to something other than `0` ?
